app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Api::V1::ReviewsController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
+
+  def create
+    user_input = JSON.parse(request.body.read)
+
```

Comment:
you should be using your review params here instead of the http request.
Line 6 is questionable on the extra line scale